src/layer.rs

```diff
@@ -484,19 +491,31 @@ impl<'a> field::Visit for SpanAttributeVisitor<'a> {
         let error_msg = value.to_string();
 
         if self.sem_conv_config.error_fields_to_exceptions {
-            self.record(Key::new(FIELD_EXCEPTION_MESSAGE).string(error_msg.clone()));
+            self.record(KeyValue::new(
+                Key::new(FIELD_EXCEPTION_MESSAGE),
+                Value::String(error_msg.clone().into()),
```

Comment:
Can we do something like `Value::from(error_msg.clone())` instead?

Reply:
I was able to use `Value::from(error_msg.clone())`